src/libs/actions/Report.js

```diff
@@ -1049,6 +1049,48 @@ function addAction(reportID, text, file) {
         .then(({reportAction}) => updateReportWithNewAction(reportID, reportAction));
 }
 
+/**
+ * Deletes a comment from the report, basically sets it as empty string
+ *
+ * @param {Number} reportID
+ * @param {Object} reportAction
+ */
+function deleteReportComment(reportID, reportAction) {
+    // Optimistic Response
+    const reportActionsToMerge = {};
+    const oldMessage = {...reportAction.message};
+    reportActionsToMerge[reportAction.sequenceNumber] = {
+        ...reportAction,
+        message: [
+            {
+                type: 'COMMENT',
```

Comment:
Looks like this should be moved to a constant to be consistent with the other actions having constants too?

Reply:
I think that is just the type of the message not the action, but I'm happy with adding those to constants as well